### DISCUSSION TITLE
Question about changing your computer's system clock without running MAS

### DISCUSSION THREAD

**Discussion by Justformas** [Monika-After-Story/MonikaModDev#9090] [QA - tech support + general] [answered]
I want to know exactly what happens in this case. Please don't try it if you don't know what will happen, I'm looking for an answer from someone who knows. First I have to describe the situation:

Suppose you last played your game on May 1st, 2022. Then you say goodbye and close your game, WITHOUT telling Monika that you're going away for awhile. Next you change your system's clock to some year way in the future, like the year 2100 or even the year 3000. Then you don't open you game again yet, but at some point before you do, you change your system's clock to May 2nd, 2022. Now when you open your game again on that day, is Monika going to think that just one day went by, or will you still lose affection? Or worse, will this somehow damage your game?

This thought occurred to me for a few reasons, one because I saw this comment: https://github.com/Monika-After-Story/MonikaModDev/issues/8212#issuecomment-964572107 which suggests that you may have to actually open your game or have it open, with the system clock changed for it to have consequences. Also because that's the way it works in some other games where real time is a factor.

**Comment by Booplicate** [comment, marked as answer]
It only affects MAS if MAS is open. Any time travel while MAS is open may corrupt event data. That said, you should avoid modifying files that MAS uses since we rely on their meta data (e.g. creation date, last mod time, etc).

**Reply by Justformas** [reply]
> It only affects MAS if MAS is open. Any time travel while MAS is open may corrupt event data. 

That's what I thought, but I wanted to know for sure. Also I realize that once you do open your game, then you can't really go back to an earlier time without consequences. Not that this specific situation I described would even be that useful, since just making sure to say that you'll be going away for awhile can negate most or all affection loss.

> That said, you should avoid modifying files that MAS uses since we rely on their meta data (e.g. creation date, last mod time, etc).

That I can agree would be too meta anyway. I haven't even really gotten into adding things the intended way yet. But anyway thank you for the answer.
